PhpAmqpLib/Connection/AbstractConnection.php

```diff
@@ -78,6 +78,9 @@ class AbstractConnection extends AbstractChannel
     /** @var int */
     protected $heartbeat;
 
+    /** @var float */
+    protected $last_frame;
```

Comment:
Agreed. Using inconsistent naming within the same file, or in this case, on the same line of code is more likely to lead to mistakes than a single large refactor. Consistency > partial standards conformance.